src/BloomExe/Book/Book.cs

```diff
@@ -679,6 +679,96 @@ public void BringBookUpToDate(IProgress progress)
 			}
 		}
 
+		class GuidAndPath
+		{
```

Comment:
Is there any advantage to being a class instead of a struct?  (I don't know, which is why I ask.)